pshops_lib/check_account_no.js

```diff
@@ -0,0 +1,28 @@
+/*
+    Function: check_account_no.js
+    Purpose: checks if the input account number appears as a PShop account in Roster
+    Status: complete
+*/
+
+module.exports = function(accnum){
+    // Authentication against Roster
+    var rosterAPI = require('ext/Roster_v1_0_2/api');
+    catchAll(function() {
```

Comment:
this structure is a bit wonky - you're defining a second function within a function that will never be called -